src/dxgi/dxgi_output.cpp

```diff
@@ -93,21 +93,42 @@ namespace dxvk {
     std::vector<DXGI_MODE_DESC> modes(modesCount);
     GetDisplayModeList(pModeToMatch->Format, 0, &modesCount, modes.data());
 
+    //filter out modes with different scaling if it was set
+    if (modeToMatch.Scaling != DXGI_MODE_SCALING_UNSPECIFIED) {
+      if (modeToMatch.Scaling != DXGI_MODE_SCALING_CENTERED) {
+        Logger::warn("DxgiOutput::FindClosestMatchingMode: DXVK currently supports only DXGI_MODE_SCALING_CENTERED. Fall back to it.");
+        modeToMatch.Scaling = DXGI_MODE_SCALING_CENTERED;
+      }
+
+      for (auto it = modes.begin(); it != modes.end();) {
+         if (it->Scaling != modeToMatch.Scaling)
```

Comment:
Why is this needed? Can't we just *ignore* the Scaling? If we're falling back to CENTERED anyway, I really don't see the point in filtering out any modes based on the scaling.

Reply:
sure, removed